tut/stream-client.md

```diff
@@ -0,0 +1,174 @@
+
+### Create an access token
+
+To access the HTTP API of the Anbox Stream Gateway, an access token is required. Each access token is associated with a service account.
+
+On the machine where Anbox Cloud Appliance is installed, create the service account by running the following command:
+
+    anbox-cloud-appliance gateway account create test
+
+The output of this command provides the access token. Make a note of this token to use when you make a request to the stream gateway API.
+
+### Create an application
+
+Follow the instructions in [How to create an application](https://discourse.ubuntu.com/t/24198) and create an application.
+
+## Set up the stream client
+Create a directory to set up the stream client:
+
+    sudo mkdir -p /srv/stream-client
+
+Create a `demo.html` file inside `/srv/stream-client` with the following content:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Streaming SDK Example</title>
+</head>
+<body>
+    <script type="module">
+    import {AnboxStream, AnboxStreamGatewayConnector} from './anbox-stream-sdk.js';
+
+    const connector = new AnboxStreamGatewayConnector({
+        // Run `juju status` to get the stream gateway IP/domain name
+        // Replace 'https://gateway.url.net' with the stream gateway IP/domain name
+        url: 'https://gateway.url.net',
+        //Use the access token created earlier as 'YOUR_AUTH_TOKEN'
+        authToken: 'YOUR_AUTH_TOKEN',
+        session: {
+            // Run `amc list` to see a list of all applications
+            // Replace `com.foo.bar` with your application name
+            app: "com.foo.bar",
+        },
+        // Set the display resolution and frame rate for the Android container
+        screen: {
+            width: 1280,
+            height: 720,
+            fps: 25,
+        }
+    });
+    // The AnboxStream class takes care of the WebRTC signaling process as well as the web browser integration
+    let stream = new AnboxStream({
+        connector: connector,
+        // targetElement is the ID of the HTML element where the SDK can attach the video
+        targetElement: "anbox-stream",
+        controls: {
+            keyboard: true
+        },
+        callbacks: {
+            error: error => {
+                console.log("AnboxStream failed: ", error);
+            }
+        }
+    });
+    stream.connect();
+    </script>
+    <!--To display the video element correctly, always specify both the height and the width attributes-->
+    <div id="anbox-stream" style="width: 100vw; height: 100vh;"></div>
+</body>
+```
+If you experience any streaming issues, you can turn on debug information by adding the following option to the `AnboxStream` which is the `main` class:
```

Comment:
```suggestion
If you experience any streaming issues, you can turn on debug information by adding the following option to the `AnboxStream`:
```

The comment about the `main` class does not seem necessary to me.

Reply:
Shouldn't we also point to our troubleshooting guides?